File: Modul-2/Soal-5/PRAK205-2410817220030-RACHELWINAYUDA.py
import math

A, B = map(float, input().split())

alas = math.sqrt(B**2 - A**2)
tinggi = A
keliling = A + B + alas
luas = 0.5 * alas * A

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

A1 = float(input())
B1 = float(input())

alas1 = math.sqrt(B1**2 - A1**2)
tinggi1 = A1
keliling1 = A1 + B1 + alas1
luas1 = 0.5 * alas1 * A1

print(f"Alas = {alas1:.0f} cm")
print(f"Tinggi = {tinggi1:.0f} cm")
print(f"Keliling = {keliling1:.0f} cm")
print(f"Luas = {luas1:.0f} cm^2")

File: Modul-2/Soal-5/PRAK205-2410817220030-RACHELWINAYUDA.c
#include <stdio.h>
#include <math.h>

int main () {
    //test case 1
    float A, B;

    scanf("%f", &A);
    scanf("%f", &B);

    float alas = sqrt(pow(B, 2) - pow(A, 2));
    float tinggi = A;
    float keliling = A + B + alas;
    float luas = 0.5 * alas * A;

    printf("Alas = %.f cm\n", alas);
    printf("Tinggi = %.f cm\n", tinggi);
    printf("Keliling = %.f cm\n", keliling);
    printf("Luas = %.f cm^2", luas);

    //test case 2
    float A1, B1;

    scanf("%f", &A1);
    scanf("%f", &B1);

    float alas1 = sqrt(pow(B1, 2) - pow(A1, 2));
    float tinggi1 = A1;
    float keliling1 = A1 + B1 + alas1;
    float luas1 = 0.5 * alas1 *A1;

    printf("Alas = %.f cm\n", alas1);
    printf("Tinggi = %.f cm\n", tinggi1);
    printf("Keliling = %.f cm\n", keliling1);
    printf("Luas = %.f cm^2", luas1);
    return 0;
}